package.json

```diff
@@ -4,45 +4,43 @@
   "description": "Nest TypeScript starter repository",
   "license": "MIT",
   "scripts": {
-    "build": "tsc -p tsconfig.build.json",
-    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
-    "start": "ts-node -r tsconfig-paths/register src/main.ts",
-    "start:dev": "nodemon",
-    "start:debug": "nodemon --config nodemon-debug.json",
-    "prestart:prod": "rimraf dist && npm run build",
+    "start": "tsc-watch --onSuccess 'node dist/main.js'",
+    "start:debug": "tsc-watch --onSuccess 'node --inspect-brk dist/main.js'",
+    "prestart:prod": "npm run build",
     "start:prod": "node dist/main.js",
-    "lint": "tslint -p tsconfig.json -c tslint.json",
     "test": "jest",
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
-    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand",
```

Comment:
Why have you removed  `-r tsconfig-paths/register -r ts-node/register` ?

Reply:
`ts-node` and `tsconfig-paths` have been replaced with `ts-watch`